Makefile

```diff
@@ -16,7 +16,7 @@ db.seed: ## Seeds the DB
 .PHONY: run-sql
 run-sql: ## Execute SQL file against the Docker instance
 	docker-compose exec -T db psql -U $(POSTGRES_USER) -d $(POSTGRES_DB) \
-		 -c "select * from worked_hours;"
+		 -c "select * from hours_worked;"
```

Comment:
Thank you for proposing this change @ShadyPenguin! I've been following the [Rails naming conventions](https://guides.rubyonrails.org/active_record_basics.html#naming-conventions), where tables are plural, columns are singular.

![naming_conventions](https://user-images.githubusercontent.com/80530/111706710-bc017b00-8810-11eb-8cf5-de17148d898e.jpg)